include/geolib/Box.h

```diff
@@ -21,14 +21,18 @@ class Box : public Shape {
 
     bool intersect(const Vector3& p) const;
 
-    Box getBoundingBox() const;
+    const Box getBoundingBox() const;
 
     void enclose(const Box& box, const Pose3D& pose);
 
     Vector3 getSize() const;
 
     Vector3 getCenter() const;
 
+    Vector3 getMin() const;
+
```

Comment:
For some reason people write double spaced code, I still fail in understanding why, you don't write double spaced reports either do you?

Reply:
Haha, keeping it aligned with what is already there.